website/docs/d/kubernetes_cluster.html.markdown

```diff
@@ -182,6 +182,11 @@ An `azure_active_directory_role_based_access_control` block exports the followin
 
 A `upgrade_settings` block exports the following:
 
+
+* `drain_timeout_in_minutes` - The amount of time in minutes to wait on eviction of pods and graceful termination per node. This eviction wait time honors waiting on pod disruption budgets. If this time is exceeded, the upgrade fails.
+
+* `node_soak_duration_in_minutes` - The amount of time in minutes to wait after draining a node and before reimaging it and moving on to next node. If not specified, the default is 0 minutes.
+
```

Comment:
Looks like there is an redundant space here and we also don't need to specify any defaults or possible values for properties in the data source docs
```suggestion
* `drain_timeout_in_minutes` - The amount of time in minutes to wait on eviction of pods and graceful termination per node. This eviction wait time honors waiting on pod disruption budgets. If this time is exceeded, the upgrade fails.

* `node_soak_duration_in_minutes` - The amount of time in minutes to wait after draining a node and before reimaging it and moving on to next node.

```